singer-io/tap-klaviyo
# TDL 18803 Add SMS events streams

# Description of change
**TDL 18803 Add SMS events streams**

- Added below SMS events streams
  - Clicked SMS
  - Consented to Receive SMS
  - Failed to Deliver SMS
  - Failed to deliver Automated Response SMS
  - Received Automated Response SMS
  - Received SMS
  - Sent SMS
  - Unsubscribed from SMS
- Create a `shared` folder for common schemas
- Updated existing scheme to use shared schema.
- Updated integration test cases to include above streams

**TDL 18962 Update schema for campaign stream**
- Update json schema for `campaign` stream

# Manual QA steps
 - Run discover mode and verify schema and metadata for each stream in the catalog.
 - Run sync mode with the state to verify bookmark logic and fields in the record. 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
